x-pack/plugins/actions/common/routes/connector/response/schemas/v1.ts

```diff
@@ -8,34 +8,89 @@
 import { schema } from '@kbn/config-schema';
 
 export const connectorResponseSchema = schema.object({
-  id: schema.string(),
-  name: schema.string(),
+  id: schema.string({
+    meta: {
+      description: 'The identifier for the connector.',
+    },
+  }),
+  name: schema.string({
+    meta: {
+      description: ' The name of the rule.',
+    },
+  }),
   config: schema.maybe(schema.recordOf(schema.string(), schema.any())),
-  connector_type_id: schema.string(),
-  is_missing_secrets: schema.maybe(schema.boolean()),
-  is_preconfigured: schema.boolean(),
-  is_deprecated: schema.boolean(),
-  is_system_action: schema.boolean(),
+  connector_type_id: schema.string({
+    meta: { description: 'The connector type identifier.' },
+  }),
+  is_missing_secrets: schema.maybe(
+    schema.boolean({ meta: { description: 'Indicates whether the connector is missing secrets.' } })
+  ),
+  is_preconfigured: schema.boolean({
+    meta: { description: 'Indicates whether the connector is preconfigured.' },
```

Comment:
In the manually maintained file we had this tidbit as well in case it's worth keeping:
```suggestion
    meta: { description: 'Indicates whether the connector is preconfigured. If true, the `config` and `is_missing_secrets` properties are omitted from the response. ' },
```